backend/common/src/main/java/ai/verta/modeldb/common/reconcilers/ReconcilerConfig.java

```diff
@@ -1,8 +1,24 @@
 package ai.verta.modeldb.common.reconcilers;
 
 public class ReconcilerConfig {
-  public int resyncPeriodSeconds = 60;
-  public int batchSize = 10;
-  public int workerCount = 10;
-  public int maxSync = 10000;
+  private int resyncPeriodSeconds = 60;
```

Comment:
@conradoverta will switching these to methods slow the reconciler down too much? Wondering since the reconciler runs so frequently.

Reply:
I'm not sure what you mean.